widgetastic_manageiq/__init__.py

```diff
@@ -5316,3 +5318,19 @@ def fill(self, values):
                             else:
                                 self.move_down_button.click()
             return True
+
+
+class SearchBox(TextInput):
+    """Overriding fill method of TextInput to send
+       enter after filling """
+
+    def fill(self, value):
```

Comment:
This is not working , after_fill is not called , hence overriding Fill.

    def after_fill(self):
        self.browser.send_keys(Keys.ENTER, self)